about.html

```diff
@@ -0,0 +1,170 @@
+                    </li>
+                    <li>
+                        <a href="about.html">About</a>
+                    </li>
+                    <li>
+                        <a href="">Join</a>
+                    </li>
+                    <li>
+                        <a href="">Sponsor</a>
+                    </li>
+                    <li>
+                        <a href="">News</a>
+                    </li>
+                    <li>
+                        <a href="">Campaign</a>
+                    </li>
+                </ul>
+            </nav>
+            <div class="container">
+                <div class="content">
+                    <p class="content-hellow">"Hellow! Music World"</p>
+                    <h1 class="main-content-heading">ZIRO MUSIC FESTIVAL NIGHT 2023</h1>
+                    <p class="content-detail">A Ziro Music Festival is a live music performance in front of an audience. 
+                        The performance may be by a single musician, sometimes then called a recital, 
+                        or by a musical ensemble, such as an orchestra, choir, or band. 
+                    </p>
+                    <p class="content-lastline">Please contact us for any further questions <br> 
+                        about zero Music Festival 2023! <br>
+                        <strong>anshumannegi777@gmail.com</strong>
+                    </p>
+                </div>
+        </div>
+        </section>
+
+        <section class="about-music" id="join">
+            <h2>Ziro Music Festival 2023</h2>
+            <p>This is an outdoor music festival which is held in Arunachal Pradesh's Ziro valley. 
+                Music lovers have found their own little heaven here! Started in 2012 by Bobby Hano and Anup Kutty, 
+                this has become a center for youngsters who love independent music.
+            </p>
+            <div class="logo">
+                <a href="">
+                    <span>Zi</span>
+                    RO
+                    <br>
+                    <SPan>Music</SPan>
+                </a>
+            </div>
+        </section>
+
+        <section class="about-past">
+            <h2>See the past Ziro Music Festivals</h2>
+            <p>Take a look at the last two Ziro Music festival.</p>
+            <div class="about-past-container">
+                <div>
+                    <img src="images/zerosecond.jpg" alt="">
+                    <div class="image-detail">
+                        <h3>2022</h3>
+                        <p>Ziro Music Festival In India</p>
+                    </div>
+                </div>
+                <div>
+                    <img src="images/Zirofirst.jpg" alt="">
+                    <div class="image-detail">
+                        <h3>2021</h3>
+                        <p>Ziro Music Festival In India</p>
+                    </div>
+                </div>
+            </div>
+        </section>
```

Comment:
This section of your application is not matching the provided design.

- [ ] The images are way too stretched that the provided design
- [ ] The heading color is not matching
- [ ] Heading has an underline in the provided design
- [ ] Also the font-family and font-weight are not matching.

| Your design  | Provided design |
| ------------- | ------------- |
| ![image](https://github.com/anshuman7negi/ConcertPage/assets/33268927/65e2f649-2bcd-42a9-992c-151f6c3612db) | ![image](https://github.com/anshuman7negi/ConcertPage/assets/33268927/80ec37ed-47d1-4afb-87d2-dbf28c6b0cc5) |

Reply:
Fixed.